slashing-protection/src/main/resources/migrations/postgresql/V2__removeNullConstraint.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE signed_blocks
```

Comment:
Similarly should just get rid of this, these changes are part of the V1 that was in the 0.2 release so think it's fine that they stay that way.

Reply:
deal